packages/components/src/components/telekom/telekom-footer-extended-navigation/telekom-footer-extended-navigation.css

```diff
@@ -43,10 +43,12 @@
   font-size: var(--telekom-typography-font-size-body);
```

Comment:
I think we can use the text style token here to make sure line height is correct:
```diff
-font-size: var(--telekom-typography-font-size-body);
+font: var(--telekom-text-style-body);
```